db/seeds.rb

```diff
@@ -57,3 +57,62 @@
   end
 
 end
+# This file should contain all the record creation needed to seed the database with its default values.
```

Comment:
This should still not be here.  Your branch is still not based off of master somehow.